### PR TITLE
Support LTS + Current versions of Django

### PR DESCRIPTION
**Pull request by cwdavies**
This change updates our supported version range to the LTS release of Django (2.2) and the current release of Django (3.1).

## Additions

- Added support for Django 3.1

## Removals

- Removed support for Django 1.11

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -20,9 +20,9 @@ The content dimension has three different views based on the grade range (elemen
 
 ## Installation for development
 
-The CR Tool will only work fully in the context of cfgov-refresh.
+The CR Tool will only work fully in the context of consumerfinance.gov.
 
-1. Clone this repository and install it locally into your cfgov-refresh environment, following [the cfgov-refresh docs for working on satellite apps](https://cfpb.github.io/cfgov-refresh/related-projects/#developing-python-packages-with-cfgov-refresh).
+1. Clone this repository and install it locally into your consumerfinance.gov environment, following [the consumerfinance.gov docs for working on satellite apps](https://cfpb.github.io/consumerfinance.gov/related-projects/#developing-python-packages-with-consumerfinance.gov).
```
Comment (Scotchester) [inline README.md:25]: The anchor in this link does not include the `.`:

```suggestion
1. Clone this repository and install it locally into your consumerfinance.gov environment, following [the consumerfinance.gov docs for working on satellite apps](https://cfpb.github.io/consumerfinance.gov/related-projects/#developing-python-packages-with-consumerfinancegov).
```

File: setup.py [hunk 2]
```diff
@@ -40,7 +40,7 @@
     },
     classifiers=[
         "Framework :: Django",
-        "Framework :: Django :: 2",
+        "Framework :: Django :: 3",
```
Comment (Scotchester) [inline setup.py:43]: This should be an addition, not a replacement.

```suggestion
        "Framework :: Django :: 2",
        "Framework :: Django :: 3",
```

Comment (cwdavies) [reply setup.py:44]: Good catch. This change has been made.

File: README.md (hunk 1 above)
Comment (cwdavies) [reply README.md]: Fixed anchor link on line 25 of README.md

**Pull request merged**